earth_enterprise/src/fusion/gecombineterrain/combineterrain.cpp

```diff
@@ -204,8 +209,13 @@ void TerrainCombiner::WriteCombinedTerrain(
     }
   }
 
-  // Create the new PacketInfo and initialize it's raw buffer.
+  END_PROFILING(calcReadSize);
+  BEGIN_PROFILING(readPackets, "read", even_path.AsString(), read_buffer_size);
```

Comment:
i would use a more descriptive name for this operation. "read" sounds very generic.